.gitmodules

```diff
@@ -7,3 +7,6 @@
 [submodule "inih"]
 	path = lib/inih
 	url = https://github.com/benhoyt/inih.git
+[submodule "lib/Middlewares/Third_Party/LwIP"]
+	path = lib/Middlewares/Third_Party/LwIP
+	url = git@github.com:prusa3d/LwIP_212.git
```

Comment:
checking out from our repo with the git protocol is ok?

Reply:
git@.. should be ok if there is ssh key added in the server, I guess so

Reply:
@joshymjose please do as @dragomirecky proposes

Reply:
for next pull request